components/atoms/TextTitle.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { useColorScheme } from 'react-native'
+import { Text, type TextProps } from 'tamagui'
+
+import getColors from '~/constants/Colors'
+
+type Props = {
+  text: string
+} & TextProps
+
+export const TextTitle = (props: Props): React.ReactElement => {
+  const colors = getColors(useColorScheme())
+  return (
+    <Text
+      {...props}
+      fontSize={14}
+      fontFamily={'JetBrainsMonoBold'}
```

Comment:
use hook [`useAppFont()`](https://github.com/NguyenVanViet2004/FE-GlowUp/blob/main/components/templates/HomeTemplate.tsx) for set fontFamily